syllabus/6-Polkadot-SDK/Polkadot-SDK-slides.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Introduction to Polkadot SDK
+description: In Introduction to Polkadot SDK
+duration: 120 mins
+---
+
+# Introduction to Polkadot SDK
+
+Note:
+
+This lecture is about learning two things:
+
+1. Polkadot
+2. Polkadot SDK
+   1. Substrate
+   2. FRAME
+   3. XCM
+
+There will be amazing modules going into further details of each. This will be very high level.
+
+---
+
+## Learning Outcomes
+
+1. Understand what Polkadot is as a developer platform.
+2. Understand the pieces of the Software Development Kit (SDK) of this platform
+
+---
+
+## Polkadot and its SDK
+
+---
+
+### 1. Polkadot
+
+Polkadot stems from Ethereum. A single threaded smart contract blockchain.
+
+Let's start with Ethereum.
+
+Let's make it sharded, while sharing security. Execution sharding, the holy grain of what ETH 2.0
+was meant to be.
+
+Let's make it heterogenous. The state transition function of these shards need NOT be the same. WASM
+is used.
+
+Let's make the deployment of those shards flexible and permissionless. We arrive at Polkadot what it
+is today.
+
+> Polkadot 1.0 is a platform to launch blockchains that can progress in parallel, whilst sharing the
+> security of Polkadot.
+
+In this architecture, we call Polkadot the Relay Chain. we call each of these blockchains a
+Parachain.
+
+Each computation unit of the Relay Chain that is capable of progressing a parachain block in
+parallel was called a Core.
+
+Polkadot initially allowed parachains to compete for deployment on Polkadot via auctioned slots.
+Winners get access to a Core for a long long time. Now, it is moving toward a more agile system where
+parachains can purchase cores on-demand.
+
+In some sense, Polkadot is moving towards a general purpose computer, exposing its Core as the
+primary primitive. Whether they are used to build a Parachain or not, it does not care. The future
+of Polkadot is the World Computer vision.
+
+### 2. How Do We Build a Parachain?
+
+Substrate and FRAME
+
+Building a blockchain is damn painful hard. This is why Polkadot has pivoted for years to build a
+modular and extensible blockchain framework called Substrate.
+
+Substrate has been used to build:
+
+1. the mighty Polkadot Relay Chain
+2. all the Polkadot Parachains to date.
+3. (less known) can be used to build any blockchain unrelated to Polkadot (solo-chain).
+
+> There is another framework involved called Cumulus, but since it has a very small footprint, we
+> will skim over it here. Cumulus augments Substrate to make it Polkadot-compatible.
```

Comment:
Cumulus is the glue that connects a Substrate solo-chain to the Polkadot Relay Chain

That analogy always worked for me :D